vembrane/ann_types.py

```diff
@@ -64,23 +64,27 @@ def __repr__(self):
 IntFloatStr = Union[int, float, str]
 
 
-def type_info(value, number="."):
+def type_info(value, number=".", field=None, record_idx=None):
     if value is None:
         return NA
+    if number == "A":
+        if len(value) != 1:
+            raise MoreThanOneAltAllele()
+        return value[0] or NA
```

Comment:
Can any of these be empty strings at some point?
If yes, you may want to retain `value[0] if value[0] is not None else NA`.

Reply:
Good catch. Forgot about that.